lib/ssl/src/ssl.erl

```diff
@@ -2013,11 +2131,14 @@ listen(Port, Options0)
 transport_accept(ListenSocket) ->
     transport_accept(ListenSocket, infinity).
 
+
+-doc(#{title => <<"Server Functions">>}).
 -doc """
-Accepts an incoming connection request on a listen socket. `ListenSocket` must
-be a socket returned from `listen/2`. The socket returned is to be passed to
-[handshake/2,3](`handshake/2`) to complete handshaking, that is, establishing
-the TLS/DTLS connection.
+Accepts an incoming connection request on a listen socket.
+
+`ListenSocket` must be a socket returned from `listen/2`. The socket
+returned is to be passed to [handshake/1,2,3](`handshake/2`) to
```

Comment:
Well probably because handshake/2 is considered the most used. But I will change it to /1 as in most cases you will see both /1 and /2 at the same time.